bio/hmmer/jackhmmer/wrapper.py

```
@@ -0,0 +1,65 @@
+"""Snakemake wrapper for jackhmmer"""
+
+__author__ = "Thomas Mulvaney, N. Tessa Pierce"
+__copyright__ = "Copyright 2025, Thomas Mulvaney, N. Tessa Pierce"
+__email__ = "mulvaney@mailbox.org"
+__license__ = "MIT"
+
+from os import path
+from snakemake.shell import shell
+
+query_file = snakemake.input.get("query")
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)